frontend/app/templates/components/footer-main.hbs

```diff
@@ -0,0 +1,47 @@
+<div class="ui center aligned container">
+  <div class="ui stackable inverted divided grid">
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Use Badgeyay</strong>
+        <a class="item" href="#" target="_blank">Sitemap</a>
+      </div>
+    </div>
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Categories</strong>
+      </div>
+    </div>
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Locations</strong>
+        <a class="item" href="#">India</a>
+        <a class="item" href="#">Singapore</a>
+        <a class="item" href="#">Berlin</a>
+        <a class="item" href="#">Mexico</a>
+        <a class="item" href="#">New York</a>
+        <a class="item" href="#">Hong Kong</a>
+      </div>
+    </div>
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Nearby Events</strong>
+      </div>
+    </div>
+    <div class="three wide column">
+      <div class="ui inverted link list">
+        <strong class="item">Follow us on</strong>
+        <a class="item" href="#" target="_blank">Facebook</a>
+        <a class="item" href="#" target="_blank">Twitter</a>
+        <a class="item" href="#" target="_blank">Github</a>
+      </div>
+    </div>
+  </div>
+  <div class="ui inverted section divider"></div>
+  <div class="ui horizontal inverted small divided link list" style="margin-top: 0;">
+    <span class="item">Copyright </span>
+    <a class="item" href="#">Imprint</a>
```

Comment:
What is the significance of `Imprint` and other things here?

Reply:
An imprint of a publisher is a trade name under which it publishes a work. We will update it as soon as we get it finalized just like open event :+1: